### DISCUSSION TITLE
Error “RTNETLINK answers: File exists” during startup

### DISCUSSION THREAD

**Discussion by dsm1212** [qdm12/gluetun#950] [Ideas]
I found this error message during startup in my container log but gluetun was running. However, my custom post-rules.txt file did not get run. Google says this error is something that happens during ifup because the gateway wasn't ready or something. I restarted and it worked without error and my post-rules were run. It's semi-serious that if ifup doesn't work and my post rules are not run I'd rather the container exit because I'm left open to ipv6 traffic. I'm not submitting a bug because I lost the log, if it happens again I'll grab it. Maybe someone can investigate the return code handling around ifup / starting openvpn? 

steve

**Comment by qdm12** [comment]
Yeah I would need the log. You could also run it with `FIREWALL_DEBUG=on` to get more logs about what commands gluetun runs. Keep me updated if this happens again :wink:
